presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionModule.java

```diff
@@ -63,7 +61,6 @@ public void configure(Binder binder)
         httpClientBinder(binder)
                 .bindHttpClient("nativeExecution", ForNativeExecutionTask.class)
                 .withConfigDefaults(config -> {
-                    config.setRequestTimeout(new Duration(10, SECONDS));
```

Comment:
Was it the case that the original version of the code didn't respect the config option and always used 10s?

Reply:
Even without the config change, we should not hard code the value in code and just use the default.

Reply:
Adding the config should solve the problem, but we should still be using the default value instead of hardcode.

Reply:
The default value from the httpClientConfig is 5min, that's also not a ideal number (too large) to use for our case IMO. Setting the default to 10s here works for most of our cases IMO.